ql/src/java/org/apache/hadoop/hive/ql/security/authorization/PrivilegeSynchronizer.java

```diff
@@ -199,57 +206,58 @@ public void run() {
+              for (String tblName : hiveClient.getAllTables(catName, dbName)) {
+                numTbl++;
+                LOG.debug("processing " + dbName + "." + tblName);
+                HiveObjectRef tableToRefresh = getObjToRefresh(HiveObjectType.TABLE, catName, dbName, tblName);
+                PrivilegeBag grantTableBag = new PrivilegeBag();
+                addGrantPrivilegesToBag(policyProvider, grantTableBag, HiveObjectType.TABLE, catName, dbName, tblName, null,
+                    authorizer);
+                hiveClient.refresh_privileges(tableToRefresh, authorizer, grantTableBag);
 
-              HiveObjectRef tableOfColumnsToRefresh = getObjToRefresh(HiveObjectType.COLUMN, dbName, tblName);
-              PrivilegeBag grantColumnBag = new PrivilegeBag();
-              Table tbl = null;
-              try {
-                tbl = hiveClient.getTable(dbName, tblName);
-                for (FieldSchema fs : tbl.getPartitionKeys()) {
-                  addGrantPrivilegesToBag(policyProvider, grantColumnBag, HiveObjectType.COLUMN,
-                          dbName, tblName, fs.getName(), authorizer);
-                }
-                for (FieldSchema fs : tbl.getSd().getCols()) {
-                  addGrantPrivilegesToBag(policyProvider, grantColumnBag, HiveObjectType.COLUMN,
-                          dbName, tblName, fs.getName(), authorizer);
+                HiveObjectRef tableOfColumnsToRefresh = getObjToRefresh(HiveObjectType.COLUMN, catName, dbName, tblName);
+                PrivilegeBag grantColumnBag = new PrivilegeBag();
+                Table tbl = null;
+                try {
+                  tbl = hiveClient.getTable(catName, dbName, tblName);
+                  for (FieldSchema fs : tbl.getPartitionKeys()) {
+                    addGrantPrivilegesToBag(policyProvider, grantColumnBag, HiveObjectType.COLUMN, catName, dbName, tblName, fs.getName(), authorizer);
+                  }
+                  for (FieldSchema fs : tbl.getSd().getCols()) {
+                    addGrantPrivilegesToBag(policyProvider, grantColumnBag, HiveObjectType.COLUMN, catName, dbName, tblName, fs.getName(), authorizer);
+                  }
+                  hiveClient.refresh_privileges(tableOfColumnsToRefresh, authorizer, grantColumnBag);
+                } catch (MetaException e) {
+                  LOG.debug("Unable to synchronize " + tblName + ":" + e.getMessage());
                 }
-                hiveClient.refresh_privileges(tableOfColumnsToRefresh, authorizer, grantColumnBag);
-              } catch (MetaException e) {
-                LOG.debug("Unable to synchronize " + tblName + ":" + e.getMessage());
               }
             }
           }
-          LOG.info("Success synchronize privilege " + policyProvider.getClass().getName() + ":" + numDb + " databases, "
-                  + numTbl + " tables");
+          LOG.info(String.format("Success synchronize privilege %s : %d catalogs, %d databases, %d tables.",
```

Comment:
Maybe?
```
LOG.info("Success synchronize privilege {} :{} catalogs, {} databases, {} tables.",numCat, numDb, numTbl);
```

Reply:
Maybe just keep as the other part with concatenation so at least it is consistens inside the file?